src/services/code-reviewer.ts

```
@@ -0,0 +1,276 @@
+import { GitHubService } from './github';
+import { AIClient } from './ai-client';
+import { AIChatMessage, DeepSeekChatCompletionResponse } from '../types/ai';
```

ESLint

[error] 'DeepSeekChatCompletionResponse' is defined but never used.

(@typescript-eslint/no-unused-vars)